### ISSUE TITLE
Please include test images

### ISSUE THREAD

**Issue by neurolabusc** [ofgulban/bvbabel#2]
Thanks for this terrific work. The permissive license will provides a nice mechanism for developers of open source tools to support data from the proprietary BrainVoyager products.

It would be great to include test images, both to validate and regression test the present code and to allow developers without BrainVoyager licenses to port your work to other languages.

There are two approaches:
 1. [nibabel](https://github.com/nipy/nibabel) includes testing scripts and validation images in their [tests](https://github.com/nipy/nibabel/tree/master/nibabel/tests/data) folder.

 2. [dcm2niix](https://github.com/rordenlab/dcm2niix) includes validation images as submodules (dcm_qa, dcm_qa_uih, dcm_qa_nih). The advantage of this approach is faster/smaller installation for end users, as they get the functionality without the bulky validation datasets.

**Comment by neurolabusc** [comment]
I tried your script [read_vmr_export_nifti.py](https://github.com/ofgulban/bvbabel/blob/08293fa0a98179ec6c53c2d850579fb264b09775/examples/read_vmr_export_nifti.py#L21) on the image `NK1_IIHC.vmr` from [here](https://brainvoyager.com/tbv/sampledata/index.html). The script successfully copies the voxel data, but sets the affine sform to the identity matrix (`np.eye()`). I am really interested in preserving the spatial transform - angulations, location of origin, and ensuring the left right are not flipped. Your work so far looks very promising.

**Comment by neurolabusc** [comment]
In addition, the [TBV_FacesHouses](https://brainvoyager.com/tbv/sampledata/index.html) dataset includes a file named `NK1.vmr` which appears to be reports a file version of 2 instead of the [documented version 4](https://support.brainvoyager.com/brainvoyager/automation-development/84-file-formats/343-developer-guide-2-6-the-format-of-vmr-files). Your `read_vmr_export_nifti.py` is unable to read this image. The pre-data header looks sensible, but the post-data header has changed.

**Comment by ofgulban** [comment]
Thanks for highlighting this issue. I am developing with using the newest data around as a priority. I will look into this old version issue.

**Comment by ofgulban** [comment]
Added an example VMR (volume image) and one of its corresponding SRF (triangular mesh surface) with 0caa086ffeedd10aeda8480554b3fd586a19eda8 .

**Comment by neurolabusc** [comment]
@ofgulban thanks! 

1. I have added preliminary [VMR, V16 and SRF support to NiiVue](https://github.com/niivue/niivue/pull/306) which will allow any device (computer, phone, table) to view these images. 
2. The current code may not correctly orient VMR files, but I can update this once you document the VMR -> NIfTI spatial transforms. I assume you can leverage [Python code](https://github.com/nipy/nibabel/blob/351ca44d52880575e022ef33c491ced6424ac037/nibabel/brainvoyager/bv_vmr.py#L152) from Thomas Emmerling.
3. Is it possible to get some example SMP format images? I would like to support [mesh overlays](https://niivue.github.io/niivue/features/mesh.layers.html) with NiiVue.

**Comment by ofgulban** [comment]
1. Cool, I checked NiiVue works really fast.
2. I will look into a general solution for VMR -> NifTI while keeping the headers correct. The issue is pulling the right information from VMR header, which is trickier than it sounds when several transformations are applied to it.
3. I have added an SMP (suraface  map) example with 223e06a together with its SRF (surface triangular mesh).

**Comment by neurolabusc** [comment]
@ofgulban thanks. I have added support for VMR, SRF and SMP to the `graph` branch of NiiVue. If you run the demo, it includes a sample BrainVoyager page that shows your SMP as an overlay for your SRF file. I tried to carefully emulate your Python code, but do tell me if my code fails for any images.

```
git clone --branch graph https://github.com/niivue/niivue.git
cd niivue
npm install
npm run demo
```

I am closing this issue, but do ping me when you have the general solution for VMR to NIfTI.
